### DISCUSSION TITLE
August 2023 Community Call

### DISCUSSION THREAD

**Discussion by heybeckyv** [ucan-wg/spec#167] [Community Call]
Sign up: https://lu.ma/ucan

Agenda

- Road to 1.0
  - @expede:
    - Updated AWAKE with MLS
    - Updated bearer token spec for security
    - Progress on 1.0.0-rc.1 specs
      - Revocation spec PR is RFR
      - Invocation & `invocation-ipld` PRs are WIP (at 50% done)
      - Delegation spec WIP (at 85% done)
  - @QuinnWilton & @bgins: updates on `rs-ucan` and `ts-ucan`

Video

[August 2023 UCAN Community Call Recording](https://youtu.be/949LBzlR1oQ?si=ToEOxaMu6oVgT22o)

Notes
- What is MLS?
  -  https://messaginglayersecurity.rocks
  -  Podcast with an intro to MLS and motivations for it: https://securitycryptographywhatever.com/2023/04/22/mls/
- UCAN as Bearer Token Specification: https://github.com/ucan-wg/ucan-http-bearer-token
- [ucan revocation v1.0.0 PR](https://github.com/ucan-wg/revocation/pull/1)
- Current UCAN milestone: https://github.com/fission-codes/Fission-Starmap/issues/75
- Full UCAN roadmap: https://starmap.site/roadmap/github.com/fission-codes/Fission-Starmap/issues/69#view=simple

Follow-Ups

**Comment by expede** [comment]
MLS — invite provenance tracking

**Comment by expede** [comment]
Alan: Unclear how to access seperate reosurces and return more UCANs in the Invocation spec

**Reply by expede** [reply]
- Permissions per task
- Often UCAN is the "on" field
